defeatMenu.h

```diff
@@ -0,0 +1,14 @@
+#ifndef DEFEATMENU_H
+#define DEFEATMENU_H
+#include <QPushButton>
+#include "gameMenu.h"
+
+//Nothing has been implemented here yet :)
+class DefeatMenu:GameMenu{
+public:
+    virtual void showMenu() override;
```

Comment:
Similar to cell members name, just show() instead of showMenu, user of the class already know it is a menu object

Reply:
Not fixed!